drivers/misc/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@
 add_subdirectory_ifdef(CONFIG_ARM_ETHOS_U ethos_u)
 add_subdirectory_ifdef(CONFIG_FT800 ft8xx)
 add_subdirectory_ifdef(CONFIG_GROVE_LCD_RGB grove_lcd_rgb)
-add_subdirectory_ifdef(CONFIG_PIO_RPI_PICO pio_rpi_pico)
+add_subdirectory_ifdef(CONFIG_RPI_PICO_PIO rpi_pico_pio)
```

Comment:
I don't think it's necessary to change the name.
It is generally named [function name]_[board name].
for example, "gpio_rpi_pico"

Reply:
Agree.
Will revisit sometime in the next few days.